contracts/ProcurementMaster.sol

```diff
@@ -34,10 +39,7 @@ contract ProcurementMaster {
         contractExecutionContract = ContractExecution(_contractExecutionAddress);
     }
 
-    // Function to deploy a new Vendor Registration Contract
-    function createVendorRegistration() public {
-        // Logic to deploy a new VendorRegistration contract
-    }
+    /// @dev removed deploy function contract as we do not need to deploy the vendor registration contract in here as all other contracts will deployed independently
```

Comment:
Small thing: I don't think we want to keep a comment about removing a function. It might be confusing for the professor/TA reviewing this. If its meant for us then we can just look at the git commit history.